content/guides/folders/single/get-lock.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Would this not be better called `get-locks`?

Reply:
Good point, modified